### DISCUSSION TITLE
problème avec maven

### DISCUSSION THREAD

**Discussion by scamassa** [heig-vd-dai-course/heig-vd-dai-course#431] [Q&A]
bonjour,
si quelqu'un pouvait m'aider à trouver d'où vient mon problème ce serait super
j'ai dû recloner notre projet pratique, et depuis que je l'ai réouvert intellij idea ne reconnaît pas le fichier pom.xml, et dit qu'il ne trouve pas non plus de projet maven.
Quelqu'un saurait ce que je dois faire? merci
Je précise que mon binôme n'a aucun problème
![Capture d'écran 2024-10-16 144854](https://github.com/user-attachments/assets/8a7bf01c-3099-4231-81a3-92cbf600702d)
![Capture d'écran 2024-10-16 144928](https://github.com/user-attachments/assets/9f9b58de-a5c3-47ab-ac8c-28f16a99089a)

**Comment by scamassa** [comment]
@ludelafo @GeraudSilvestri

**Reply by ludelafo** [reply]
Est-ce que cette discussion est en rapport avec la dernière que vous avez ouverte ici <https://github.com/orgs/heig-vd-dai-course/discussions/452> ?
